activerecord/lib/active_record/attribute_methods/write.rb

```diff
@@ -17,7 +17,7 @@ def define_method_attribute=(name)
             generated_attribute_methods.module_eval <<-STR, __FILE__, __LINE__ + 1
               def __temp__#{safe_name}=(value)
                 name = ::ActiveRecord::AttributeMethods::AttrNames::ATTR_#{safe_name}
```

Comment:
As we're in the vicinity and squeezing performance on this method... I wonder how this constant lookup compares to `#{name.inspect}.freeze`